core/templates/Polyfills.ts

```diff
@@ -18,6 +18,7 @@
 
 import 'globalthis/auto';
 import 'proxy-polyfill';
+import '@webcomponents/custom-elements';
```

Comment:
Oh interesting, why import it here? We don't seem to use customElements here. Could you add before/after videos of testing?

Reply:
@kevintab95 Since this is a polyfill, I imported it in the polyfills file so as to keep all polyfill imports in one place. Plus this file will be imported by the root component and hence will be available to all files which use custom elements.